### ISSUE TITLE
docker run -it -p 5000:5000 max-facial-age-estimator fails 

### ISSUE THREAD

**Issue by ptitzler** [IBM/MAX-Facial-Age-Estimator#26]
Follow the instructions in https://github.com/IBM/MAX-Facial-Age-Estimator#run-locally.
When you try to launch the Docker image the following error is displayed:

```
$ docker run -it -p 5000:5000 max-facial-age-estimator
Using TensorFlow backend.
2019-04-22 21:47:08.146488: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
2019-04-22 21:47:08.146561: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2019-04-22 21:47:08.146577: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2019-04-22 21:47:08.146588: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2019-04-22 21:47:08.146598: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
Traceback (most recent call last):
  File "app.py", line 2, in <module>
    from api import ModelMetadataAPI, ModelPredictAPI
  File "/workspace/api/__init__.py", line 2, in <module>
    from .predict import ModelPredictAPI  # noqa
  File "/workspace/api/predict.py", line 21, in <module>
    class ModelPredictAPI(PredictAPI):
  File "/workspace/api/predict.py", line 23, in ModelPredictAPI
    model_wrapper = ModelWrapper()
  File "/workspace/core/model.py", line 50, in __init__
    self.detector = MTCNN()
  File "/opt/conda/lib/python3.6/site-packages/mtcnn/mtcnn.py", line 195, in __init__
    weights = np.load(weights_file).item()
  File "/opt/conda/lib/python3.6/site-packages/numpy/lib/npyio.py", line 447, in load
    pickle_kwargs=pickle_kwargs)
  File "/opt/conda/lib/python3.6/site-packages/numpy/lib/format.py", line 692, in read_array
    raise ValueError("Object arrays cannot be loaded when "
ValueError: Object arrays cannot be loaded when allow_pickle=False
```

**Comment by bdwyer2** [comment]
I ran into this same issue an hour ago when building locally but it looks like Dockerhub is building successfully.

**Comment by I-Hong** [comment]
This was fixed by https://github.com/IBM/MAX-Facial-Age-Estimator/pull/28#issue-272538890

**Comment by MLnick** [comment]
Fixed by #24
